zkmemory/src/commitment/kzg.rs

```diff
@@ -136,17 +140,16 @@ where
     // This is made compatible with the Fr endianess
     fn be_bytes_to_field(&self, bytes: &mut [u8]) -> Fr {
```

Comment:
```rust
    // Convert 32 bytes number to field elements
    // This is made compatible with the Fr endianess
    fn be_bytes_to_field(&self, bytes: &mut [u8]) -> Fr {
        bytes.reverse();
        let inner = [0, 8, 16, 24].map(|i| u64::from_le_bytes(bytes[i..i + 8].try_into().unwrap()));
        Fr::from_raw(inner)
    }
```

`bytes` was borrowed as `mutable` then `.reverse()` it will change the data in the next call.